CHANGELOG

```diff
@@ -1,3 +1,14 @@
+snowplow-unified 0.2.0 (2024-02-XX)
+---------------------------------------
+## Summary
+XXX
+
+## Features
+- Add new passthrough aggregations to the views, sessions, and users table, enabled using `snowplow__view/session/user_aggregations`
```

Comment:
Are you confident in that the '/' in the var name won't break something down the line?